content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -63,6 +63,24 @@ resources:
           keys:
             - name: key1
               secret: YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXoxMjM0NTY=
+  - resources:
+      - events
+    providers:
+      - identity: {} # do not encrypt events even though  *.* is specified below
```

Comment:
Can you add a separate section below to explain how someone can opt-out something while wildcard has been enabled?